src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String EMPTY_CHARACTER = " ";
+    private static final String COMMA = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        createFile(toFileName);
+        String writeTo = readFromFile(fromFileName);
+        writeToFile(toFileName, writeTo);
+    }
 
+    private void createFile(String fileName) {
+        File file = new File(fileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create a file.");
+        }
+    }
```

Comment:
do we need this method here?

Reply:
If i don't create this method, variable toFileName has grey color. I wrote this method in getStatistic, but if I follow the logic of "Common mistakes" I have to create a separate method for this.